bowtie/_cli.py

```diff
@@ -1648,12 +1657,22 @@ async def info(
 
     async for _, each in start():
         metadata = [(k, v) for k, v in each.info.serializable().items() if v]
+        if show_versions:
+            versions = await each.get_versions()
+            metadata.append(
+                (
+                    "versions",
+                    list(versions) if versions else [each.info.version],
```

Comment:
This fallback logic (to return just the one version) seems like it belongs in `get_versions`?

Reply:
Fixed this!